### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by TSG405**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -21,7 +21,7 @@
 /mysql/ @ruslanbes
 /node.js/ @pbachman
 /php/ @ruslanbes
-/python/ @tik9 @Amsal1
+/python/ @tik9 @Amsal1 @tsg405
```
Comment (Ebazhanov) [inline CODEOWNERS:24]: @TSG405 please also do not forget to add youself to the Readme.md inside column Responsible, thanks

Comment (Ebazhanov) [reply CODEOWNERS:24]: do not ignore my message please, if you want to be Codeowners you have to add your self here is well, thanks
image
and make sure you can answer qeustions.

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
left comment

File: CODEOWNERS (hunk 1 above)
Comment (TSG405) [reply CODEOWNERS:24]: Okay, gotcha

**Review by Ebazhanov — APPROVED** [review]
still do not know why do want to be there? This file is outdated was for different reason. We probably need to remove it, thanks

**Pull request merged**